### PR TITLE
Fix up links to tutorials

### PR DESCRIPTION
**Pull request by barrymcgee**
## Done

Fix up links to tutorials

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
-  Sanity check link changes

### CODE REVIEW

File: templates/templates/_tag_manager.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         ga('linker:autoLink', ['conjure-up.io', 'login.ubuntu.com', 'www.ubuntu.com',
         'ubuntu.com', 'insights.ubuntu.com', 'developer.ubuntu.com', 'cn.ubuntu.com',
         'design.ubuntu.com', 'maas.io', 'canonical.com', 'landscape.canonical.com',
-        'pages.ubuntu.com', 'tutorials.ubuntu.com', 'docs.ubuntu.com']);
+        'pages.ubuntu.com', 'www.ubuntu.com/tutorials', 'docs.ubuntu.com']);
```
Comment (pmahnke) [inline templates/templates/_tag_manager.html]:
```suggestion
        'pages.ubuntu.com', 'docs.ubuntu.com']);
```

not required

File: templates/templates/_navigation-community-h.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h4 class="p-heading--four is-dense u-hide--small">Learn</h4>
           <h4 class="p-muted-heading u-hide--medium u-hide--large">Learn</h4>
           <p class="p-p--small u-hide--small">Tutorials cover a wide range of topics. Learn or teach something new to Ubuntu users.</p>
           <ul class="p-text-list--small is-bordered">
-            <li class="p-list__item"><a href="http://tutorials.ubuntu.com/">Ubuntu Tutorials</a></li>
+            <li class="p-list__item"><a href="https://www.ubuntu.com/tutorials/">Ubuntu Tutorials</a></li>
```
Comment (pmahnke) [inline templates/templates/_navigation-community-h.html]:
```suggestion
            <li class="p-list__item"><a href="/tutorials">Ubuntu Tutorials</a></li>
```
we don't need the domain

File: templates/legal/websites.html [hunk 3]
```diff
@@ -116,7 +116,7 @@ <h1>Canonical websites</h1>
           <a href="https://snapcraft.io">snapcraft.io</a>
         </li>
         <li class="p-list__item">
-          <a href="https://tutorials.ubuntu.com">tutorials.ubuntu.com</a>
+          <a href="https://www.ubuntu.com/tutorials">www.ubuntu.com/tutorials</a>
```
Comment (pmahnke) [inline templates/legal/websites.html]:
```suggestion
          <a href="/tutorials">www.ubuntu.com/tutorials</a>
```
we don't need the domain on the actual href

**Comment by barrymcgee** [comment]
D'oh... now amended and ready for re-review.

**Pull request merged**